### PR TITLE
feat(replays): Show seconds in the tooltip when hovering Replay Details header

### PR DESCRIPTION
**Pull request by ryan953**
Here's the tooltip with seconds displayed now:

Screen Shot 2022-09-26 at 11 22 44 AM

Because the tooltip has a max-width, the content wraps which doesn't look the best. For that reason I made it the new format an option, **not the default**.

For exmple, if we changed the default then the Activity list would also start to wrap, see before & after:
| Current Tooltip | Display seconds in tooltip (not implemented) |
| ------------- | ------------- |
| Screen Shot 2022-09-26 at 11 29 37 AM  | Screen Shot 2022-09-26 at 11 25 33 AM  |

### CODE REVIEW

File: static/app/components/timeSince.tsx [hunk 1]
```diff
@@ -82,7 +84,10 @@ function TimeSince({
   const dateObj = getDateObj(date);
   const user = ConfigStore.get('user');
   const options = user ? user.options : null;
-  const format = options?.clock24Hours ? 'MMMM D, YYYY HH:mm z' : 'LLL z';
+  const tooltipFormat = tooltipShowSeconds
+    ? 'MMMM D, YYYY h:mm:ss A z'
+    : 'MMMM D, YYYY h:mm A z';
+  const format = options?.clock24Hours ? 'MMMM D, YYYY HH:mm z' : tooltipFormat;
```
Comment (ryan953) [inline static/app/components/timeSince.tsx:90]: https://momentjs.com/docs/#/displaying/

**Comment by evanpurkhiser** [comment]
Should we maybe use short month name so it doesn’t wrap?

**Review by billyvg — COMMENTED** [review]
I almost wonder if we should accept arbitrary formatting here. Maybe not since our date formatting should be as consistent as possible, and Replays is a one-off that would benefit from displaying seconds.

The wrapping is not great -- since this was an issue I brought up randomly, let's wait for post DEX to merge. (I think ideally we would increase the max-width?).

**Comment by ryan953** [comment]
I was thinking that increasing the max-width is the easiest. It seems like it's a random number at 225px right now. 
But increasing it ~whenever isn't the best strategy.

**Pull request closed without merging**